OAuth/ResourceOwner/FacebookResourceOwner.php

```diff
@@ -102,14 +112,15 @@ protected function configureOptions(OptionsResolver $resolver)
         parent::configureOptions($resolver);
 
         $resolver->setDefaults(array(
-            'authorization_url' => 'https://www.facebook.com/v2.8/dialog/oauth',
-            'access_token_url' => 'https://graph.facebook.com/v2.8/oauth/access_token',
-            'revoke_token_url' => 'https://graph.facebook.com/v2.8/me/permissions',
-            'infos_url' => 'https://graph.facebook.com/v2.8/me?fields=first_name,last_name,name,email',
+            'authorization_url' => 'https://www.facebook.com/v3.0/dialog/oauth',
```

Comment:
Changing the API should be done in a separate commit / pull request.

Reply:
@guillaume-ro-fr Any idea why you want to change the API? The profile picture should work with v2.8 as well.